src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,563 @@
+  {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group pf-m-icon-group"}}
+    {{> inline-edit-action-save inline-edit-action-save--IsIcon="true"}}
+    {{> inline-edit-action-cancel inline-edit-action-cancel--IsIcon="true"}}
+  {{/inline-edit-group}}
+{{/inline-edit}}
+```
+
+```hbs title=Single-inline-edit-(default)
+{{#> inline-edit inline-edit--id="single-inline-edit-example" inline-edit--type="form"}}
+  {{#> inline-edit-group}}
+    {{#> inline-edit-value inline-edit-value--IsLabel="true"}}
+      Static value
+    {{/inline-edit-value}}
+    {{#> inline-edit-toggle}}
+      {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'id="' inline-edit--id '-edit-button" aria-label="Edit" aria-labelledby="' inline-edit--id '-edit-button ' inline-edit--id '-label"')}}
+        <i class="fas fa-pencil-alt" aria-hidden="true" />
+      {{/button}}
+    {{/inline-edit-toggle}}
+  {{/inline-edit-group}}
+  {{#> inline-edit-group}}
+    {{#> inline-edit-input}}
+      {{> form-control controlType="input" input="true" form-control--attribute='type="text" value="Static value" aria-label="Editable text input"'}}
+    {{/inline-edit-input}}
+    {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group pf-m-icon-group"}}
+      {{> inline-edit-action-save inline-edit-action-save--IsIcon="true"}}
+      {{> inline-edit-action-cancel inline-edit-action-cancel--IsIcon="true"}}
+    {{/inline-edit-group}}
+  {{/inline-edit-group}}
+{{/inline-edit}}
+```
+
+```hbs title=Single-inline-edit-(active)
+{{#> inline-edit inline-edit--id="single-editable-example" inline-edit--type="form" inline-edit--modifier="pf-m-inline-editable"}}
+  {{#> inline-edit-group}}
+    {{#> inline-edit-value inline-edit-value--IsLabel="true"}}
+      Static value
+    {{/inline-edit-value}}
+    {{#> inline-edit-toggle}}
+      {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'id="' inline-edit--id '-edit-button" aria-label="Edit" aria-labelledby="' inline-edit--id '-edit-button ' inline-edit--id '-label"')}}
+        <i class="fas fa-pencil-alt" aria-hidden="true" />
+      {{/button}}
+    {{/inline-edit-toggle}}
+  {{/inline-edit-group}}
+  {{#> inline-edit-group}}
+    {{#> inline-edit-input}}
+      {{> form-control controlType="input" input="true" form-control--attribute='type="text" value="Static value" aria-label="Editable text input"'}}
+    {{/inline-edit-input}}
+    {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group pf-m-icon-group"}}
+      {{> inline-edit-action-save inline-edit-action-save--IsIcon="true"}}
+      {{> inline-edit-action-cancel inline-edit-action-cancel--IsIcon="true"}}
+    {{/inline-edit-group}}
+  {{/inline-edit-group}}
+{{/inline-edit}}
+```
+
+```hbs title=Single-inline-edit-with-label-(default)
+{{#> inline-edit inline-edit--type="form" inline-edit--id="single-inline-edit-with-label-example"}}
+  {{#> inline-edit-group}}
```

Comment:
why is the label and the action button inside a group but then the value is outside of the group? I thought that the value always sits inside of a group with the action button because that is what is directly being edited. Could you explain a little more :)

Reply:
Sure. The `-group` is now arbitrary and performs a basic function that `display: flex`s elements by default. The idea being that instead of having to call a layout, you can (optionally) use the `.pf-c-inline-edit__group` to provide basic alignment, but it's not necessary. `-groups` no longer hide/show, rather the elements within now hide/show.